bamboo/use-working-directory.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+## script moves from <current> at <GIT_SHA> to /uncached/cumulus and checks out <GIT_SHA>
+set -ex
+
+GIT_SHA=$(git rev-parse HEAD)
+export GIT_SHA
```

Comment:
Is there any value in logging the GIT_SHA here? If you made it this far maybe not. I doubt there's danger of having the incorrect SHA checked out.

Reply:
using -ex *does* log it, the output in the logs here looks like 


```
error	21-Apr-2022 13:21:41	++ set -ex
error	21-Apr-2022 13:21:41	+++ git rev-parse HEAD
error	21-Apr-2022 13:21:41	++ GIT_SHA=f10ddb7a2b23c6aa442d1356a97dd6868cc203fe
error	21-Apr-2022 13:21:41	++ export GIT_SHA
```